addons/website/models/ir_model.py

```diff
@@ -38,3 +39,14 @@ def get_website_meta(self):
         # dummy version of 'get_website_meta' above; this is a graceful fallback
         # for models that don't inherit from 'website.seo.metadata'
         return {}
+
+    def _get_lang_base(self):
+        self.ensure_one()
+        website = False
+        if 'website_id' in self._fields:
+            website = self.website_id
+        if not website:
+            website = ir_http.get_request_website()
```

Comment:
```suggestion
            website = self.env['ir.http'].get_request_website()
```
also remove the unnecessary import

Reply:
I think it is a function of ir_http.py instead of a method of `ir.http`.
https://github.com/odoo/odoo/blob/0227e6dc2f9ab155e458a0f46f946e0f1fb4f55f/addons/website/models/ir_http.py#L56

Reply:
oh right it is, and you cannot from-import it since it is patched in tests, alrighty then